mappings/net/minecraft/block/DyedCarpetBlock.mapping

```diff
@@ -1,6 +1,12 @@
 CLASS net/minecraft/class_5815 net/minecraft/block/DyedCarpetBlock
+	COMMENT A carpet that has an associated carpet color for {@linkplain net.minecraft.entity.passive.LlamaEntity llamas}.
 	FIELD field_28715 dyeColor Lnet/minecraft/class_1767;
 	METHOD <init> (Lnet/minecraft/class_1767;Lnet/minecraft/class_4970$class_2251;)V
 		ARG 1 dyeColor
+			COMMENT the color of this carpet when worn by a {@linkplain net.minecraft.entity.passive.LlamaEntity llama}
 		ARG 2 settings
 	METHOD method_33635 getDyeColor ()Lnet/minecraft/class_1767;
+		COMMENT {@return the color of this carpet when worn by a {@linkplain net.minecraft.entity.passive.LlamaEntity llama}}
```

Comment:
It's also the colour of the carpet in general, see #2468

Reply:
This is the only instance in which the color is actually used. I agree that the method should be named `getColor`, but in vanilla it is used solely for llama carpet colors.

Reply:
imo that llama usage should go to apiNote